front/src/applications/stdcmV2/components/StdcmConfig.tsx

```diff
@@ -71,7 +69,11 @@ const StdcmConfig = ({
   const origin = useSelector(getOrigin);
   const destination = useSelector(getDestination);
 
-  const { pathfindingState } = usePathfinding(undefined, pathProperties);
+  const [pathfindingProperties, setPathfindingProperties] = useState<
+    ManageTrainSchedulePathProperties | undefined
+  >(undefined);
```

Comment:
You don't need to specify that the undefined here ;) 
```suggestion
    ManageTrainSchedulePathProperties
  >();
```